packages/blade/src/tokens/global/elevation/elevation.native.ts

```diff
@@ -0,0 +1,88 @@
+import type { ElevationWithColorModes } from './types';
+
+export const elevation: ElevationWithColorModes = {
+  onLight: {
+    none: {
+      elevation: 0,
+      shadowColor: '',
+      shadowOpacity: 0,
+      shadowRadius: 0,
+      shadowOffset: {
+        width: 0,
+        height: 0,
+      },
+    },
+    lowRaised: {
+      elevation: 4,
+      shadowColor: 'hsla(217, 56%, 17%, 0.64)',
```

Comment:
Possible to pick this up from the global color palette instead of hardcoding?

Reply:
https://github.com/razorpay/blade/pull/1178#discussion_r1201661863